lib/test-requirements.txt

```diff
@@ -35,5 +35,5 @@ pixelmatch>=0.3.0
 pytest-xdist
 pytest-rerunfailures
 pytest-github-actions-annotate-failures
-pytest-benchmark==5.1.0
-pytest-repeat==0.9.3
+pytest-benchmark>=5.1.0
+pytest-repeat>=0.9.3
```

Comment:
Drive-by change to only set the lower bound version to be consistent with how we handle other test dependencies in python.